src/http/routing/route-registry.ts

```
@@ -678,7 +678,8 @@ export class RouteRegistry {
   }
 
   private getMiddlewareName(middleware: Type<unknown> | object): string {
-    return typeof middleware === 'function' ? middleware.name : middleware.constructor.name;
+    if (!middleware) return 'UnknownMiddleware';
+    return typeof middleware === 'function' ? middleware.name : (middleware.constructor?.name || 'AnonymousMiddleware');
```

ESLint

[error] Replace `·?·middleware.name·:·(middleware.constructor?.name·||·'AnonymousMiddleware')` with `⏎······?·middleware.name⏎······:·middleware.constructor?.name·||·'AnonymousMiddleware'`

(prettier/prettier)